theme/scripts/patronSlider.ts

```diff
@@ -0,0 +1,161 @@
+import getNextSibling from "./utils/getNextSibling";
+
+export default (function () {
+  const PatronSlider = {
+    SELECTORS: {
+      slider: "[data-patron-slider]",
+      handle: ".noUi-handle",
+      pips: ".noUi-marker",
+      pipsValue: ".noUi-value",
+      patronTierLink: "[data-patron-tier-link]",
+      patronTierDescription: "[data-patron-tier-description]",
+      patronTierName: "[data-patron-tier-name]",
+    },
+    START_VALUE: 15,
+    MONTHLY: "Monthly",
+    PATRON_TIERS: {
+      "5": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7795091",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public.",
+        name: "Community Patron",
+      },
+      "10": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7795092",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Patron",
+      },
+      "15": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849151",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also support our ongoing programming at Index.",
+        name: "Supporter Patron+",
+      },
+      "20": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7836923",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Patron",
+      },
+      "25": {
+        href: "https://www.patreon.com/join/index_space/checkout?rid=7849152",
+        description:
+          "You'll get access to our community Discord, Arena, Time Exchange, patron-only events, and our archived talks earlier than the public. You'll also help sustain our programming at Index.",
+        name: "Sustaining Patron+",
+      },
+    },
+
+    SLIDER: null,
+
+    init() {
+      // Initialize noUiSlider instance.
+      PatronSlider.SLIDER = rangeSlider.create(
+        document.querySelector(PatronSlider.SELECTORS.slider),
+        {
+          // Start with the slider at this value.
+          start: [PatronSlider.START_VALUE],
+          // Add this many notches along the slider.
+          step: Object.keys(PatronSlider.PATRON_TIERS).length,
+          range: {
+            min: 5,
+            max: 25,
+          },
+          // Options for notches along the slider, called "pips"
+          pips: {
+            mode: rangeSlider.PipsMode.Steps,
+            filter: (value) => {
+              // If a slider value is not in our list of values,
```

Comment:
we could use array includes here to make things a bit simpler. Also, let's store these tiers in a constant at the top of our file (outside the module)

```
const Tiers = [5, 10, 15, 20, 25];
filter: (value) => Tiers.includes(value)
```